shuup/front/apps/customer_information/templates/shuup/customer_information/addressbook/index.jinja

```diff
@@ -0,0 +1,71 @@
+{% extends "shuup/front/dashboard/dashboard.jinja" %}
+{% from "shuup/customer_information/macros/address.jinja" import render_address_forms, render_address_forms_js %}
+{% from "shuup/customer_information/macros/buttons.jinja" import user_account_button, save_changes_button %}
+{% from "shuup/customer_information/macros/contact.jinja" import render_company_contact_form %}
+
+{% block title %}{% trans %}Address Book{% endtrans %}{% endblock %}
+{% block content_title %}{% trans %}Address Book{% endtrans %}{% endblock %}
+
+{% block breadcrumb %}
+    {% call macros.breadcrumb() %}
+        {{ macros.render_breadcrumb_item("/", _("Home")) }}
+        {{ macros.render_breadcrumb_item(url("shuup:dashboard"), _("Dashboard")) }}
+        {{ macros.render_breadcrumb_item(None,  _("Address Book"), True) }}
+    {% endcall %}
+{% endblock %}
+
+
+{% macro render_address(addr) %}
+    {% set region = (addr.region_code or addr.region) %}
+    <p><b>{% trans %}Name{% endtrans %}:</b> {{ addr.name }}</p>
+    <p><b>{% trans %}Email{% endtrans %}:</b> {{ addr.email or "Not specified" }}</p>
+    <p><b>{% trans %}Phone{% endtrans %}:</b> {{ addr.phone or "Not specified" }}</p>
+    {% if addr.postal_code and addr.city and region %}
+        <p><b>{% trans %}Address{% endtrans %}:</b><br>{{ addr.street or "Not specified" }}</p>
+        <p>{{ addr.postal_code }} {{ addr.city }}, {{ region }}</p>
+    {% else %}
+        <p><b>{% trans %}Street{% endtrans %}:</b> {{ addr.street or "Not specified" }}</p>
+        <p><b>{% trans %}Postal code{% endtrans %}:</b> {{ addr.postal_code or "Not specified" }}</p>
+        <p><b>{% trans %}City{% endtrans %}:</b> {{ addr.city or "Not specified" }}</p>
+        <p><b>{% trans %}Region{% endtrans %}:</b> {{ region }}</p>
+    {% endif %}
+{% endmacro %}
+
+{% block dashboard_content %}
+<div class="user-account-address">
+    <h1>{% trans %}Default Addresses{% endtrans %}
+    <a class="btn btn-primary pull-right" role="button" href="{{ url("shuup:customer_edit") }}">{% trans %}Edit{% endtrans %}</a>
+    </h1>
+    <div class="col-md-6">
+        {% call content_block(_("Default Billing Address")) %}
+            {% set addr = customer.default_billing_address %}
+            {{ render_address(addr) }}
```

Comment:
Can you just pass the customer.default_billing_address without setting any variables? Also the `render_address` fails little in case the address is not set.